woocommerce/woocommerce
# Related Products: hide unusable options from Inspector Controls

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





@jarekmorawski brought to my attention Related Products offer some controls that are unusable:
- Advanced Filters (valid in products (Beta) but not here)
- Upgrade Notice - Product Collection doesn't support Related Products case and clicking the upgrade does nothing.

This PR hides both sections.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Editor -> Single Product template
2. Add Related Products block if there isn't one
3. Focus on "Related Products Controls"
4. Expected: there's no "Advanced Filters" or "Upgrade Notice" sections in Inspector Controls (check screenshot below)

Before | After
-- | --
![Screen Shot 2024-05-27 at 14 56 41 PM]
<img width="282" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/291c664b-9351-472e-9e12-abf909ab62da"> | <img width="282" alt="image" src="https://github.com/woocommerce/woocommerce/assets/20098064/a8109746-4528-48e2-a91d-ee1edfb8c7ea">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
